R/psychsyn_critval.R

```diff
@@ -23,5 +23,7 @@ psychsyn_critval <- function(x, anto = FALSE) {
   else {
   correlations <- correlations[order(correlations$Freq, decreasing = FALSE),]
   }
+  names(correlations) <- c("Var1","Var2","Correlation")
```

Comment:
```suggestion
  names(correlations) <- c("var1","var2","cor")
```

Small edit for consistency with output of the psychsyn function